docs/2.7.0/bin/setup-sphinx-source-tree

```diff
@@ -16,26 +16,22 @@ prefix=$(basename $(realpath $DIR/..))
 
 mkdir -p $SPHINX_DIR/source/canton $SPHINX_DIR/source/daml-finance
 tar xf $DOWNLOAD_DIR/sphinx-source-tree-$RELEASE_TAG.tar.gz -C $SPHINX_DIR --strip-components=1
-tar xf $DOWNLOAD_DIR/canton-doc-sources-$CANTON_RELEASE_TAG.tar.gz -C $SPHINX_DIR/source/canton
-rm $SPHINX_DIR/source/canton/canton-refs.rst
+tar xf $DOWNLOAD_DIR/canton-doc-sources-$CANTON_RELEASE_TAG.tar.gz -C $SPHINX_DIR/source/canton ./includes # only extract the includes folder
 tar xf $DOWNLOAD_DIR/daml-finance-doc-sources-$DAML_FINANCE_RELEASE_TAG.tar.gz -C $SPHINX_DIR/source/daml-finance
 
+cp -r $DIR/../docs/canton/* $SPHINX_DIR/source/canton
 cp $SPHINX_DIR/source/canton/exts/canton_enterprise_only.py $SPHINX_DIR/configs/static/
 
 # Rewrite Canton's absolute references.
 find $SPHINX_DIR/source/canton -type f -print0 | while IFS= read -r -d '' file
 do
-    sed -i 's|include:: /substitution.hrst|include:: /canton/substitution.hrst|g ; s|image:: /images|image:: /canton/images|g' $file
```

Comment:
ℹ️ This `sed`s had no effect because `substitution.hrst` doesn't exist anymore. And `/images` has been replaced with `./images` which is relative to where the `rst` is located.